tests/std/tests/VSO_0180469_ptr_cat/test.cpp

```diff
@@ -239,6 +239,18 @@ void test_case_Equal_memcmp_is_safe() {
     test_case_Equal_memcmp_is_safe_helper<false, const Elem1*, const volatile Elem2*, equal_to<>>();
     test_case_Equal_memcmp_is_safe_helper<false, const volatile Elem1*, const volatile Elem2*, equal_to<>>();
 
+    // Top level const should work

```

Comment:
What does "work" mean here?

Reply:
I moved it above and changed it to `// Top level const should not change the answer`